web/wwwroot/css/site.css

```diff
@@ -58,6 +58,28 @@ input.invalid {
   background-color: #04AA6D;
 }
 
-.tables{
+/*.tables{

```

Comment:
What's the reason that this is commented out? 🙂

Reply:
I think this is leftover from a different branch :)